modules/ocf_stats/manifests/prometheus.pp

```diff
@@ -1,8 +1,25 @@
 # prometheus daemon config
 class ocf_stats::prometheus {
+  file {
+    '/usr/local/bin/gen-prometheus-nodes':
+      source => 'puppet:///modules/ocf_stats/prometheus/gen-prometheus-nodes',
+      mode   => '0755';
+  }
+
+  cron { 'gen-prometheus-nodes':
+    command => '/usr/local/bin/gen-prometheus-nodes /var/local/prometheus-nodes.json',
+    minute  => '03',
```

Comment:
I'm guessing this was taken from `gen-munin-nodes`:
https://github.com/ocf/puppet/blob/26109d27c9c19d1b683c9a32af428a0a01ebe51c/modules/ocf_stats/manifests/munin.pp#L27-L35

The `03` minute argument there existed because munin runs every 5 minutes, and generating the nodes while munin was going to run gave problems (rt#4712), as mentioned in the comment, so I'd probably make this on the hour or something instead to make it more regular and predictable.

Reply:
Done. As an aside, maybe `gen-munin-nodes` should do the same atomic file replace that's used in my script?

Reply:
Yeah, that's probably a better idea than what I did.